organization/_posts/2022/2022-01-20-coordination.md

```diff
@@ -0,0 +1,214 @@
+---
+title: "HSF Coordination Meeting #221, 20 January 2022"
+layout: plain_toc
+---
+
+# Attending
+
+Present/Contributing:
+Alaettin Serhan Mete,
+Ale Di Girolamo,
+Alexander Moreno,
+Allison Hall,
+Andrea Valassi,
+Andreas Salzburger,
+Ben Morgan,
+Benedikt Hegner,
+David DeMuth,
+David Lange,
+Dorothea vom Bruch,
+Eduardo Rodrigues,
+Efe Yazgan,
+Gordon Watts,
+Graeme Stewart,
+Jim Pivarski,
+Jin Huang,
+Josh McFayden,
+Kevin Pedro,
+Krzysztof Genser,
+Kyle Knoepfel,
+Liz Sexton-Kennedy,
+Marc Paterno,
+Mark Neubauer,
+Markus Diefenthaler,
+Matti Kortelainen,
+Michael Wilkinson,
+Michel Jouvin,
+Michel Villanueva,
+Nicole Skidmore,
+Oksana Shadura,
+Paul Laycock,
+Pere Mato,
+Serhan Mete,
+Stefan Roiser,
+Stephan Hageboeck,
+Sudhir Malik,
+Torre Wenaus,
+Valentin Volkl,
+Wouter Deconinck
+
+Apologies/Contributing: 
+
+
+## News, general matters, announcements
+
+**Happy New Year!**
+
+### HSF Talks
+
+#### Lepton-Photon
+
+Graeme gave an overview [plenary talk](https://indico.cern.ch/event/949705/contributions/4575453/) about Software and Computing R&D in Lepton-Photon last week - thanks to all who make suggestions for improvements. Discussion focused on use of accelerators and HPCs (perenial topics of interest!).
+
+## Working Group Updates
+
+### General
+
+A big welcome to all of out incoming conveners for 2022 and many thanks to all of our continuing conveners.
+
+Many thanks to TJ Khoo (Analysis); Chris Jones and Attila Krasznahorkay (Frameworks); Andrea Valassi (Generators); Ben Krikler (PyHEP); David Lange (Reconstruction); Mircho Rozodov (Developer Tools); and Meirin Oan Evans (Training) for their work in 2021 and, in many cases, for quite a number of years before that. It’s very much appreciated!
+
+After this meeting outgoing conveners will be liberated from the mailing list, but we look forward to your continuing involvement in HSF and the WGs.
+
```

Comment:
Let's add a line here about the WGs having uploaded slides to <https://indico.cern.ch/event/1096580/> - otherwise the minutes read strangely